src/dropdown.js

```diff
@@ -18,6 +18,7 @@ export type DropdownOptions = {
   rotate?: boolean,
   style?: { [string]: string },
   item?: DropdownItemOptions,
+  parent?: HtmlElement,
```

Comment:
`HTMLElement`? https://github.com/facebook/flow/blob/v0.102.0/lib/dom.js#L1633

Reply:
@yuku good catch!

Fixed in 84ba469